README.md

```diff
@@ -8,31 +8,43 @@ The UI is bundled with the [Marathon](https://github.com/mesosphere/marathon) pa
 Please note that issues are disabled for this repository. Please feel free to open an issue on the
 [issues page on the main Marathon Repository](https://github.com/mesosphere/marathon/issues?q=is%3Aopen+is%3Aissue+label%3Agui).
 
-#### Setup
+#### Prequisites
+
+Please make sure you've installed and properly configured the following software:
 
-1. Install Mesos and Marathon (follow the [tutorial here](https://mesosphere.github.io/marathon/docs/))
-2. Setup a CORS proxy on your machine to proxy the UI requests to your running Marathon instance (e.g. via [Corsproxy](https://www.npmjs.com/package/corsproxy))
-3. Install [Node 5](https://nodejs.org/en/blog/release/v5.0.0/) and [NPM](https://npmjs.org/)
+* [Docker 1.9](https://www.docker.com/)
+* [Node 5](https://nodejs.org/en/blog/release/v5.0.0/) including [NPM](https://npmjs.org/)
+
+#### Setup
 
-4. Install dev dependencies
+##### 1. Install all dependencies
 
         npm install
         npm install -g gulp
 
-5. Override development configuration
+##### 2. Configure your hosts
+
+If you're not using something like [dnsdock](https://github.com/tonistiigi/dnsdock) or [dinghy](https://github.com/codekitchen/dinghy) (OS X) for easy container discovery/access, please configure your're hosts as follows:
+
+      mesos-master.docker current.docker.machine.ip
+      mesos-slave.docker  current.docker.machine.ip
+      marathon.docker     current.docker.machine.ip
+
+*Use `$ docker-machine ip $DOCKER_MACHINE_NAME` to get the current docker machine ip and add those lines to your `etc/hosts` configuration.*
```

Comment:
I think the `.` notation is misleading with words - how about `127.0.0.0` as a placeholder instead?